### PR TITLE
update quick start guide for creating workload using enclave-cc

### PR DESCRIPTION
**Pull request by hairongchen**
update workload deployment using SGX hardware mode with encrypted and cosign signed image

Signed-off-by: hairongchen <hairong.chen@intel.com>

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -187,39 +187,45 @@ root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/contain
 ## Creating a sample Coco workload using enclave-cc
 
 Following the previous example that used the `kata` runtime class, we setup a sample *hello world*
-workload using the `enclave-cc` runtime class for process based TEEs. The deployment below assumes
-the simulated SGX mode build (`ccruntime-enclave-cc-sim.yaml`) is installed by the operator. With that,
-the workload can be deployed on a non-TEE system too.
+workload with an encrypted and cosign signed image using the `enclave-cc` runtime class for process based TEEs. The deployment below assumes
+the hardware SGX mode build (`ccruntime-enclave-cc.yaml`) is installed by the operator. To try on a non-TEE system, please use (`ccruntime-enclave-cc-sim.yaml`).
```
Comment (mythi) [inline quickstart.md:191]: one additional update for v0.3.0 is the we moved to kustomize based deployment which means these `yaml`s should not be used directly. Can you make the changes in this PR as well?

**Review by mythi — COMMENTED** [review]
@hairongchen thanks! One comment about kustomize.

File: quickstart.md (hunk 1 above)
Comment (hairongchen) [reply quickstart.md]: thanks Mikko, updated with kustomize  style.

**Review by mythi — APPROVED** [review]
LGTM

**Comment by hairongchen** [comment]
@dcmiddle we need one more review and approve

**Review by dcmiddle — APPROVED** [review]

**Review by fidencio — APPROVED** [review]
lgtm, thanks @hairongchen!

**Pull request merged**